### PR TITLE
[APP-15119] - Add createIntegrationHelpers

### PR DESCRIPTION
**Pull request by geovannimp**

### CODE REVIEW

File: packages/integration-sdk-core/tools/generateTypeboxSchemas.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { TObject } from '@sinclair/typebox';
```
Comment (geovannimp) [inline packages/integration-sdk-core/tools/generateTypeboxSchemas.ts:1]: # Just ignore from here to the end. This must be moved to data-model repo.

**Review by jzolo22 — APPROVED** [review]

File: packages/integration-sdk-core/package.json [hunk 2]
```diff
@@ -23,8 +23,9 @@
     "prepack": "yarn build:dist"
   },
   "dependencies": {
-    "@jupiterone/data-model": "^0.57.0",
+    "@jupiterone/data-model": "^0.61.3",
     "@jupiterone/integration-sdk-entity-validator": "^12.7.0",
+    "@sinclair/typebox": "^0.32.30",
```
Comment (eyadmba) [inline packages/integration-sdk-core/package.json:28]: Can typebox be a dev dependency?

File: tsconfig.base.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
     "lib": ["ES2022", "DOM"],
     "module": "commonjs",
     "moduleResolution": "node",
+    "resolveJsonModule": true,
```
Comment (eyadmba) [inline tsconfig.base.json:9]: What did we need this for?

**Review by eyadmba — DISMISSED** [review]

File: packages/integration-sdk-core/package.json (hunk 2 above)
Comment (geovannimp) [reply packages/integration-sdk-core/package.json:28]: No because we need it in runtime to generate the schema.

File: tsconfig.base.json (hunk 3 above)
Comment (geovannimp) [reply tsconfig.base.json:9]: For some reason something changed in the data model and it cannot import the json schemas without this anymore :cry:

**Review by jzolo22 — DISMISSED** [review]

**Review by jzolo22 — APPROVED** [review]

**Review by eyadmba — APPROVED** [review]
🔥

**Pull request merged**